### PR TITLE
Added Polygon, Polyline. Line fixed.

### PR DESCRIPTION
**Pull request by KvanTTT**

### CODE REVIEW

File: src/Perspex.Controls/Shapes/Polyline.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+namespace Perspex.Controls.Shapes
+{
+    public class Polyline: Shape
+    {
+        public static readonly PerspexProperty<IList<Point>> PointsProperty =
+            PerspexProperty.Register<Polyline, IList<Point>>("Points");
+
+        public Polyline()
+        {
+            StrokeThickness = 1; // Default Thickness
```
Comment (kekekeks) [inline src/Perspex.Controls/Shapes/Polyline.cs]: `PerspexProperty.Register` allows you to specify default value

File: src/Perspex.SceneGraph/Media/PolylineGeometry.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                }
+
+                return new Rect(xMin, yMin, xMax - xMin, yMax - yMin);
+            }
+        }
+
+        /// <inheritdoc/>
+        public override Geometry Clone()
+        {
+            return new PolylineGeometry(new List<Point>(_points), _isFilled);
```
Comment (kekekeks) [inline src/Perspex.SceneGraph/Media/PolylineGeometry.cs]: Platform impl has `Clone` method that can implement some copy-on-write optimizations

File: src/Perspex.Controls/Shapes/Polyline.cs (hunk 1 above)
Comment (KvanTTT) [inline src/Perspex.Controls/Shapes/Polyline.cs]: ```StrokeThickness``` is already defined in Shape:
```C#
public static readonly PerspexProperty<double> StrokeThicknessProperty =
            PerspexProperty.Register<Shape, double>("StrokeThickness");
```
So, how can I specify default value?

Comment (kekekeks) [inline src/Perspex.Controls/Shapes/Polyline.cs]: https://github.com/Perspex/Perspex/blob/5957d33d75bff3834b27808601322544eb6382fc/src/Perspex.Base/PerspexProperty.cs#L638

Comment (KvanTTT) [inline src/Perspex.Controls/Shapes/Polyline.cs]: I should rewrite this by the following way?
```C#
static Polyline()
{
    StrokeThicknessProperty.OverrideDefaultValue<Polyline>(1); // Default Thickness
}
```

Comment (kekekeks) [inline src/Perspex.Controls/Shapes/Polyline.cs]: +

File: src/Perspex.SceneGraph/Media/PolylineGeometry.cs (hunk 2 above)
Comment (KvanTTT) [inline src/Perspex.SceneGraph/Media/PolylineGeometry.cs]: Something like this? But it's not working correctly:
```C#
private PolylineGeometry(IGeometryImpl impl)
{
    PlatformImpl = impl;
}

/// <inheritdoc/>
public override Geometry Clone()
{
    return new PolylineGeometry(((IStreamGeometryImpl)PlatformImpl).Clone());
}
```

**Pull request merged**